common/views/components/Paginator/Paginator.js

```diff
@@ -97,40 +105,40 @@ const Paginator = ({
         })}
       >
         {showPortal && <div id="sort-select-portal"></div>}
-        {prevLink && prev && (
-          <Space as="span" h={{ size: 'm', properties: ['margin-right'] }}>
-            <Control
-              link={prevLink}
-              clickHandler={event => {
-                onPageChange(event, prev);
-              }}
-              type="light"
-              extraClasses={classNames({
-                'icon--180': true,
-              })}
-              icon="arrow"
-              text={`Previous (page ${prev})`}
-            />
-          </Space>
-        )}
-
-        <span>
-          Page {currentPage} of {totalPages}
-        </span>
-
-        {nextLink && next && (
-          <Space as="span" h={{ size: 'm', properties: ['margin-left'] }}>
-            <Control
-              link={nextLink}
-              clickHandler={event => {
-                onPageChange(event, next);
-              }}
-              type="light"
-              icon="arrow"
-              text={`Next (page ${next})`}
-            />
-          </Space>
-        )}
+        <PaginatorWrapper hideMobilePaging={hideMobilePaging}>
```

Comment:
Could we not just use the [`useScreenSize` hook](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/hooks/useWindowSize.js) and remove the DOM completely?

Why would we rather hide it via css?

Reply:
That is also a solution i could do, I was considering non javascript version as well. Should we leverage on both ?

Reply:
Good point, the hook is for JS.